### DISCUSSION TITLE
Error while selecting UUID: 22P02 - invalid input syntax for type bigint

### DISCUSSION THREAD

**Discussion by johannesmcb** [supabase/supabase#11336] [Questions]
Hello Guys,

I have a table "resources" with an ID (type: uuid) and i'm trying to fetch it in NextJS server component via supabase-server:

`const { data: resource, error } = await supabase.from('resources').select('*, appointment(*)').eq('id', params.id);`

The error response:
`{
  code: '22P02',
  details: null,
  hint: null,
  message: 'invalid input syntax for type bigint: "20c21f68-c750-443e-b52b-28e42ee44b90"'
}`

Any ideas on this?

`resources: {
          id: string (UUIDv4)
          ....
}`

**Comment by steve-chavez** [comment]
Hello,

This is likely a bug within the supabase UI, when you change a column type(or rename table) sometimes the UI doesn't make the change.

Restarting the db has been reported as a workaround on discord.

cc @GaryAustin1

**Reply by GaryAustin1** [reply]
@johannesmcb Can you confirm you changed the the column type...  And if so, did you encounter any errors?

**Reply by johannesmcb** [reply]
@GaryAustin1  @steve-chavez  

That's funny... First I tried to change the ID from INT to UUID. Then I got severeal errors on the UI in Supabase.. I dropped the table and recreated it from scratch.

Same error.

**Reply by GaryAustin1** [reply]
I've gone thru an id as int, change to uuid, get error UI error, delete table in UI, create new table in UI with same name and UUID type id column.  

It works.  This Postgres 15 and latest instance software.

So something more to the equation.

**Reply by johannesmcb** [reply]
Ok, that's weird. Thanks for your help so far!
